### PR TITLE
test: remove top-level code in test-indexing.R

### PR DESCRIPTION
**Pull request by maelle**
@szhorvat if `am()` and `mm()` had longer names, what would those be? I'd like to replace them with longer, more informative names.

Removing top-level code in test files help make each test more self-contained, therefore easier to re-run interactively when it fails (as happens in #1080)

Part of #1082

### CODE REVIEW

**Comment by maelle** [comment]
@szhorvat based on your input I'll rename the two functions (if you agree!) then once it's merged I can rebase #1080 and remove the test that's failing there (it's failing because it's using edges, which we now expect to not work).

**Comment by szhorvat** [comment]
I'm not sure what these names are referring to.

File: tests/testthat/helper-indexing.R
Comment (maelle) [inline tests/testthat/helper-indexing.R:6]: am() is a helper function in tests. maybe it could be called as_matrix? @szhorvat

File: tests/testthat/helper-indexing.R
Comment (maelle) [inline tests/testthat/helper-indexing.R:1]: this is mm(). It takes a vector and makes into a matrix. vector_to_matrix?

File: tests/testthat/helper-indexing.R
Comment (szhorvat) [reply tests/testthat/helper-indexing.R:6]: How about `canonicalize_matrix`? (Since you asked. But use anything you like.)

File: tests/testthat/helper-indexing.R
Comment (szhorvat) [reply tests/testthat/helper-indexing.R:1]: How about `vector_to_square_matrix`?

File: tests/testthat/helper-indexing.R
Comment (maelle) [reply tests/testthat/helper-indexing.R]: Thank you!

File: tests/testthat/helper-indexing.R
Comment (maelle) [reply tests/testthat/helper-indexing.R]: Thanks!

**Review by szhorvat — APPROVED** [review]
Just to dismiss the yellow "review requested" bar.

**Pull request merged**
